File: tests/query/highlights/java/constructs.java
class Box {
//    ^ @type
    boolean method(int count) {
//  ^ @type.builtin
//          ^ @function.method
        String s = "a\tb";
//      ^ @type
//                   ^ @constant.character.escape
        return helper(count);
//             ^ @function.method
    }
}

File: tests/query/highlights/typescript/constructs.ts
interface Shape { size: number; }
//        ^ @type
//                ^ @variable.other.member
//                      ^ @type.builtin
class Box {
//    ^ @type
  method(count: number): void {
//^ @function.method
    obj.doThing();
//      ^ @function.method
  }
}
